includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -376,6 +376,21 @@ public function init_fields() {
 		if ( ! get_option( 'job_manager_enable_remote_position' ) ) {
 			unset( $this->fields['job']['remote_position'] );
 		}
+
+		if ( get_option( 'job_manager_enable_scheduled_listings' ) ) {
+			$field_type = version_compare( JOB_MANAGER_VERSION, '1.30.0', '>=' ) ? 'date' : 'text';
+
+			$this->fields['job']['job_schedule_listing'] = [
+				'label'       => __( 'Schedule date', 'wp-job-manager' ),
+				'description' => __( 'Optionally schedule a date for this listing to go public.', 'wp-job-manager' ),
```

Comment:
Not 100% sure about what the best labeling would be here, but the others are nouns and 'schedule date' is more of an action. So something like `Scheduled Date` or `Publish Date`?

```suggestion
				'label'       => __( 'Scheduled Date', 'wp-job-manager' ),
				'description' => __( 'Optionally set the date when this listing will be published.', 'wp-job-manager' ),
```